README.md

```diff
@@ -14,6 +14,13 @@ All of the following repositories should be checked out in adjacent directories:
 * [notifications-credentials](https://github.com/alphagov/notifications-credentials.git)
 
 You can just run the `./clone-repos.sh` script if you have git+ssh configureed. This will importantly also run `make generate-version-file` in each repo, which is needed by most of the app dockerfiles.
+### Pre Requisites
+
+If you haven't run the clone repos above as you already had them downloaded, just make sure to generate the versions for these repos
+
+`cd notifications-admin` and run `make generate-version-file` to get a vesrion file for the local build of notifications-admin
+`cd notfications-api` and `make generate-version-file`
+`cd notfications-template-preview` and `make generate-version-file`
```

Comment:
You can also just safely re-run the clone-repos script I think. Might be easier to recommend that rather than manually cd/generating?